contrib/tsdev.sh

```diff
@@ -1,29 +1,88 @@
+	echo
+	echo "Run commands to manage resources on your Timesketch development docker container"
+	echo
+	echo "Options:"
+	echo "  -h, --help        Display this help message"
+	echo
+	echo "Commands:"
+	echo "  build-api-cli     Build and install API and CLI clients"
+	echo "  celery            Start a celery worker"
+	echo "  logs              Follow docker container logs "
+	echo "  shell             Open a shell in the docker container"
+	echo "  test              Execute run_tests.py --coverage"
+	echo "  vue-build         Build the vue frontend"
+	echo "  vue-dev           Serve the vue frontend"
+	echo "  vue-install-deps  Install vue frontend dependencies"
+	echo "  vue-test          Test the vue frontend"
+	echo "  web               Start the web frontend bound to port 5000"
+	echo
+	echo "Examples:"
+	echo "  tsdev.sh logs"
+	echo "  tsdev.sh test"
+	exit 0
+}
 
-if [ $1 == "web" ]; then
-  docker exec -it $CONTAINER_ID gunicorn --reload -b 0.0.0.0:5000 --log-level debug --capture-output --timeout 600 timesketch.wsgi:application
-elif [ $1 == "celery" ]; then
-  docker exec -it $CONTAINER_ID celery -A timesketch.lib.tasks worker --loglevel=info
-elif [ $1 == "vue-install-deps" ]; then
-  docker exec -it $CONTAINER_ID yarn install --cwd=/usr/local/src/timesketch/timesketch/$frontend
-elif [ $1 == "vue-dev" ]; then
-  docker exec -it $CONTAINER_ID yarn run --cwd=/usr/local/src/timesketch/timesketch/$frontend serve
-elif [ $1 == "vue-build" ]; then
-  docker exec -it $CONTAINER_ID yarn run --cwd=/usr/local/src/timesketch/timesketch/$frontend build
-elif [ $1 == "vue-test" ]; then
-  docker exec -it $CONTAINER_ID yarn run --cwd=/usr/local/src/timesketch/timesketch/frontend-ng test
-elif [ $1 == "test" ]; then
-  docker exec -w /usr/local/src/timesketch -it $CONTAINER_ID python3 run_tests.py --coverage
-elif [ $1 == "shell" ]; then
-  docker exec -it $CONTAINER_ID /bin/bash
-elif [ $1 == "logs" ]; then
-  docker logs -f $CONTAINER_ID
-elif [ $1 == "build_api_cli" ]; then
-    docker exec -w '/usr/local/src/timesketch/cli_client/python' -it $CONTAINER_ID python3 setup.py build
-    docker exec -w '/usr/local/src/timesketch/cli_client/python' -it $CONTAINER_ID python3 setup.py install
-    docker exec -w '/usr/local/src/timesketch/api_client/python' -it $CONTAINER_ID python3 setup.py build
-    docker exec -w '/usr/local/src/timesketch/api_client/python' -it $CONTAINER_ID python3 setup.py install
+# Display the help message if there is no command or if the help option is present
+if [ $# -eq 0 ] || [ "$1" == "-h" ]  || [ "$1" == "--help" ]; then
+	help
 fi
+
+# Use "sudo" for docker commands if sudoless docker is not enabled
+# "docker version" will return "true" if sudoless docker is configured
+if docker version >/dev/null 2>&1; then
+	s=""
+else
+	s="sudo"
+	sudo -v
+fi
+
+# Set variables required to execute commands
+CONTAINER_ID="$("$s" docker container list --filter name='timesketch-dev' --quiet)"
```

Comment:
Removing the quotes here. With the quotes it will fail to execute the command.

```suggestion
CONTAINER_ID="$($s docker container list --filter name='timesketch-dev' --quiet)"
```